src/spatial/views/framework_map_views.py

```diff
@@ -0,0 +1,41 @@
+import os
+import logging
+from flask import render_template, jsonify
+import requests
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+class AQIColorView:
+    def __init__(self):
+        self.access_token = os.getenv('AIRQO_API_TOKEN')
+        self.mapbox_token = os.getenv('MAPBOX_ACCESS_TOKEN')
```

Comment:
Hi @wabinyai , thanks for this.

Just curious as to why we are using Mapbox for this use case. Considering AirQo Mapbox account is purely commercial and paid for by AirQo, do we envision the sharing of the AirQo MAPBOX account with external people (organisations)?